addons/web/static/src/legacy/js/views/basic/basic_view.js

```diff
@@ -329,6 +329,16 @@ var BasicView = AbstractView.extend({
             }
         }
 
+        if ('allow_drag_drop_create' in attrs) {
+            /**
+             * used to configure, for a field, whether we want to use the front-end
+             * logic to get the group value. (i.e. with kanban drag&drop and quickCreate features)
+             * if false, those features will be disabled for the current field. Supported types are:
+             * char, boolean, many2one, selection, date, datetime
+             */ 
+            attrs.allow_drag_drop_create = JSON.parse(attrs.allow_drag_drop_create);
```

Comment:
in JS the convention is to camelCase variables, so
```js
attrs.allowDragDropCreate = JSON.parse(attrs.allow_drag_drop_create);
```
Still not sure about the name though